symbology/create-symbols-from-web-styles/src/main/java/com/esri/samples/create_symbols_from_web_styles/CreateSymbolsFromWebStylesSample.java

```
@@ -0,0 +1,319 @@
+import com.esri.arcgisruntime.symbology.Symbol;
+import com.esri.arcgisruntime.symbology.SymbolStyle;
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer;
+
+public class CreateSymbolsFromWebStylesSample extends Application {
+
+  private MapView mapView;
+  private GridPane legendGridPane;
+  private GridPane headerGridPane;
+  private SymbolStyle symbolStyle;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Create Symbols From Web Styles Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the light gray basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_LIGHT_GRAY);
+
+      // set an initial reference scale on the map for controlling symbol size
+      map.setReferenceScale(100000);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set a viewpoint on the map view
+      mapView.setViewpoint(new Viewpoint(34.28301, -118.44186, 7000));
+
+      // create a feature layer from a service
+      FeatureLayer featureLayer = new FeatureLayer(new ServiceFeatureTable(
+        "http://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0"));
+
+      // add the feature layer to the map's operational layers
+      map.getOperationalLayers().add(featureLayer);
+
+      // display an error if the feature layer fails to load
+      featureLayer.addDoneLoadingListener(() -> {
```

I'm not sure how I feel about these add done loading listeners for if something has failed to load. I seem to remember this was something that was brought in a few releases ago to show the loadable pattern consistenly in our samples though.  It just seems like they get in the way a bit of the flow of the sample. If they have to stay due to a previous decision, could we separate out a method that takes a `Loadable` and call that method in place of this, and the later SymbolStyle loading check?

---

@Rachael-E Yeah I get your point - we don't have to have them, I guess it's the balance with making the sample flow better vs silently failing and showing how to avoid that - but I guess if we know the data works in our sample we can leave them out... Guess there are other samples (or tutorials etc) for learning loading patterns and that's not necessarily the point of this sample? Shall I just remove these?